src/Permissions/Permission.php

```diff
@@ -1,56 +1,42 @@
 <?php
 
+declare(strict_types=1);
+
 namespace LaravelDoctrine\ACL\Permissions;
 
 use Doctrine\ORM\Mapping as ORM;
```

Comment:
I do not agree with Doctrine's mapping of attributes to the Mapping namespace.  They should be in `Attribute`